source/features/repo-contained-header.css

```diff
@@ -0,0 +1,25 @@
+.pagehead .UnderlineNav-item:focus,
+.pagehead .UnderlineNav-item:hover {
+	border-bottom-color: var(--repo-nav-border-bottom-color-hover, #2ea44f80) !important;
+}
```

Comment:
I don't think this matches GitHub's style very much.

Reply:
I agree, I was looking for something with more contrast against the light gray border. How about https://primer.style/css/support/variables#$border-gray-darker? I would be okay with removing this declaration if need be.

Reply:
> I would be okay with removing this declaration if need be.

I'm fine with the `$border-gray-darker`, but let's see how maintainers think about this.